test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Member.Tests.ps1

```diff
@@ -256,7 +256,7 @@ Describe "Get-Member DRT Unit Tests" -Tags "CI" {
     Context "Verify Get-Member with other parameters" {
         It 'works with View Parameter' {
             $results = [xml]'<a>some text</a>' | Get-Member -view adapted
-            $results.Length | Should Be 38
```

Comment:
I don't think this is a good or useful test - a better test would sample for some expected members.